src/certificates/certificatesApi.ts

```diff
@@ -64,12 +64,24 @@ export class CertificatesApi {
     }
 
     private extendCertificate(iamCert: iamCertificate, done: (error: any, certificate: any) => any) {
+        if (iamCert.device_execution_mode === 1) {
```

Comment:
👍  My code to fix it look exactly the same. I only have if else

Reply:
:)